packages/react/checkbox/src/Checkbox.stories.tsx

```diff
@@ -91,14 +91,34 @@ export const WithinForm = () => {
   );
 };
 
-export const Animated = () => (
-  <Label>
-    Label{' '}
-    <Checkbox className={rootClass}>
-      <CheckboxIndicator className={`${indicatorClass} ${animatedIndicatorClass}`} />
-    </Checkbox>
-  </Label>
-);
+export const Animated = () => {
+  const [checked, setChecked] = React.useState<boolean | 'indeterminate'>('indeterminate');
+
+  return (
+    <>
+      <p>
+        <Checkbox
+          className={rootClass}
+          checked={checked}
+          onCheckedChange={(event) => setChecked(event.target.checked)}
+        >
+          <CheckboxIndicator className={animatedIndicatorClass} />
+        </Checkbox>
+      </p>
+
+      <button
+        type="button"
+        onClick={() =>
+          setChecked((prevIsChecked) =>
+            prevIsChecked === 'indeterminate' ? false : 'indeterminate'
+          )
```

Comment:
Is this actually a "fix" for what we saw?

Reply:
Nope, this is just a copy/paste of the `indeterminate` story so that I could show an animated example of all states.

Reply:
So do we actually have an issue with checkbox animation or not? Or was it resolved by removing the rafs?

Reply:
There wasn't an issue with the checkbox animation, it looked broken because when the checkbox was `unchecked` it had no height so it wasn't visible even though it *was* fading out.

I've added height to `&[data-state="unchecked"]` to fix that.